docs/operate/config-reference.mdx

```diff
@@ -622,10 +622,10 @@ Connections to non-EVM chains only support a single RPC url. If more than one RP
 </TabItem>
 
 <TabItem value="env" label="As Env">
-```sh
-export HYP_CHAINS_${CHAIN_NAME}_RPCCONSENSUSTYPE="single"
-export HYP_CHAINS_ETHEREUM_RPCCONSENSUSTYPE="single"
-```
+  ```sh 
+  export HYP_CHAINS_${CHAIN_NAME}_RPCCONSENSUSTYPE="single" 
```

Comment:
i assume the spaces at the ends of these lines are all from prettier? a bit surprised by them but nbd if they're auto-added. If they're intentionally added and not by prettier, curious what the benefit is?